app/mbm/templates/mbm/montreal.html

```diff
@@ -0,0 +1,110 @@
+{% extends "mbm/base.html" %}
+{% load leaflet_tags %}
+{% load static %}
+
+{% block title %}{{ view.title }}{% endblock %}
+
+{% block extra_css %}
+  {% leaflet_css %}
+  <link rel="stylesheet" href="{% static 'css/spin.css' %}" />
+{% endblock %}
+
+{% block body %}
+  <div class="row">
+    <div id="controls-container" class="col-12 col-md-3">
+      <div class="container-fluid m-auto form-group">
+        <form id="input-elements" class="hideable collapse multi-collapse">
+          <div class="form-group">
+            <label for="source" class="sr-only">Start address</label>
+            <input id="source" class="form-control my-2" name="source" placeholder="Start address"></input>
+          </div>
+          <div class="form-group">
+            <label for="target" class="sr-only">End address</label>
+            <input id="target" class="form-control my-2" name="target" placeholder="End address"></input>
+          </div>
+          <div class="form-group">
+            <button id="submit" type="submit" class="btn btn-primary btn-block mt-1">Search</button>
+            <button id="reset-search" type="reset" class="btn btn-secondary btn-block mb-2">Reset search</button>
+          </div>
+          <div class="form-group">
+            <div>
+              <input type="checkbox" id="enable-v2" name="enable-v2" checked />
```

Comment:
This new file is mostly copied from `index.html`, other than turning on `enable-v2` by default here and passing the city name to `new App()`.